clients/src/test/java/org/apache/kafka/clients/producer/internals/TransactionManagerTest.java

```diff
@@ -202,7 +203,7 @@ public boolean matches(AbstractRequest body) {
 
         sender.run(time.milliseconds());  // Send AddOffsetsRequest
         assertTrue(transactionManager.hasPendingOffsetCommits());  // We should now have created and queued the offset commit request.
-        assertFalse(addOffsetsResult.isDone());
+        assertTrue(addOffsetsResult.isCompleted());
```

Comment:
Jason got rid of the `FutureTransactionalResult` and uses the `TransactionalRequestResult` directly, which has different underlying method names.